Bugfixes/datadocs ui

roblim opened the pull request:
Addresses:
- https://app.asana.com/0/1119694864223484/1162583857360089/f
- https://app.asana.com/0/1119694864223484/1161644922195953/f
- https://app.asana.com/0/1119694864223484/1145663702660542/f
- https://app.asana.com/0/1119694864223484/1133008169429685/f

Before:
<img width="1675" alt="Screenshot 2020-02-18 16 21 48" src="https://user-images.githubusercontent.com/3468435/75896299-bbf03200-5deb-11ea-8979-f11a799b21a9.png">
<img width="498" alt="Screenshot 2020-03-03 23 34 00" src="https://user-images.githubusercontent.com/3468435/75896330-c8748a80-5deb-11ea-8b09-70da30ad3bba.png">
<img width="495" alt="Screenshot 2020-03-03 23 38 14" src="https://user-images.githubusercontent.com/3468435/75896363-d4f8e300-5deb-11ea-9dfe-fac44fbdfb1d.png">

After:
<img width="1584" alt="Screenshot 2020-03-04 07 44 06" src="https://user-images.githubusercontent.com/3468435/75896428-f0fc8480-5deb-11ea-9f21-9fe054310a99.png">
<img width="493" alt="Screenshot 2020-03-03 23 35 13" src="https://user-images.githubusercontent.com/3468435/75896470-ff4aa080-5deb-11ea-9c2a-95a210bd695d.png">
<img width="500" alt="Screenshot 2020-03-03 23 37 37" src="https://user-images.githubusercontent.com/3468435/75896509-0c678f80-5dec-11ea-84d0-508c60790821.png">

roblim committed:
Crop glossary scroller gif

roblim committed:
Change modal headings to h4

roblim committed:
Nix extraneous ge_info

roblim committed:
Adjust breakpoints

roblim committed:
Nix extraneous ge_info

roblim committed:
Adjust breakpoints for section padding

roblim committed:
Make buttons flow vertically on small screen

roblim committed:
Fix index page link table padding

roblim committed:
Fix padding

roblim committed:
Adjust padding

roblim committed:
Hide breadcrumbs on xs viewport size

roblim committed:
Nix l/r padding

roblim committed:
Increase left padding

roblim committed:
Nix link table horizontal padding

roblim committed:
Update test snapshots

pep8speaks commented:
Thank you for submitting to Great Expectations!





* In the file [`tests/render/test_default_jinja_view.py`](https://github.com/great-expectations/great_expectations/blob/f4e357b218e8ea61907516933cea35472dd916b4/tests/render/test_default_jinja_view.py):

> [Line 129:7](https://github.com/great-expectations/great_expectations/blob/f4e357b218e8ea61907516933cea35472dd916b4/tests/render/test_default_jinja_view.py#L129): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace

This linter comment has been updated based on the most recent commit.

##### Comment last updated at 2020-03-05 18:00:01 UTC

roblim committed:
Merge branch 'develop' into bugfixes/datadocs-ui

roblim committed:
Merge branch 'develop' into bugfixes/datadocs-ui

Aylr reviewed (changes requested):
This looks great - I love the before and after screens! QQ: is the deletion of the scroller gif intentional?

Aylr committed:
Merge branch 'develop' into bugfixes/datadocs-ui

roblim committed:
Merge branch 'develop' into bugfixes/datadocs-ui

roblim commented:
> This looks great - I love the before and after screens! QQ: is the deletion of the scroller gif intentional?

@Aylr - it's still there. Just deleted old version and replaced with new cropped version.

roblim committed:
Add to changelog

roblim committed:
Merge branch 'bugfixes/datadocs-ui' of github.com:great-expectations/great_expectations into bugfixes/datadocs-ui
